Assets/Source/Components/Environment/ParallaxComponent.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Assets.Source.Components.Base;
+using Assets.Source.Constants;
+using UnityEngine;
+
+namespace Assets.Source.Components.Environment
+{
+    public class ParallaxComponent : ComponentBase
```

Comment:
Its confusing having ParallaxComponent and ParallaxBehavior, rename those my vote